docs/guides/es2k/enabling-comms-channel.md

```diff
@@ -0,0 +1,132 @@
+# Enable COMMS channel
+
+COMMS channel enablement is required to establish communication between
+different compute complexes on IPU. Here, we will be giving an example
+to enable communication between ACC and Host. This allows P4Runtime client
```

Comment:
we can also mention any p4runtime client can run from HOST and communicate to server on ACC.
Example p4runtime shell or K8S runtime clients..